dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -78,23 +75,28 @@ public ContentTypeResource(final ContentTypeHelper contentletHelper, final WebRe
 	@NoCache
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-	public final Response createType(@Context final HttpServletRequest req, final String json)
+	public final Response createType(@Context final HttpServletRequest req, final ContentTypeForm form)
 			throws DotDataException, DotSecurityException {
 		final InitDataObject initData = this.webResource.init(null, true, req, true, null);
 		final User user = initData.getUser();
 
 		Response response = null;
 
 		try {
-			final List<ContentType> typesToSave = new JsonContentTypeTransformer(json).asList();
```

Comment:
Print more logging in order to know what inputs are coming.

Reply:
in this method both exceptions are the same:
```
 catch (DotStateException e) {

			response = ExceptionMapperUtil.createResponse(null, "Content-type is not valid ("+ e.getMessage() +")");

		} catch (DotDataException e) {

			response = ExceptionMapperUtil.createResponse(null, "Content-type is not valid ("+ e.getMessage() +")");

		}
```
You could use (DotStateException | DotDataException e) instead

Reply:
@freddyucv tiny thing but good review